opentelemetry-api/src/opentelemetry/metrics/aggregation.py

```diff
@@ -11,3 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
```

Comment:
Heh, wonder why this appears as a file rename (from `distributedcontext/__init__.py`)

Reply:
Good catch. Will fix this.